flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/semantic_tokens/source/main.bal

```diff
@@ -0,0 +1,35 @@
+import ballerina/io;
+
+// Module-level variables for semantic token tests
+int myVar = 10;
+string name = "John";
+boolean flag = true;
+int var1 = 1;
+int var2 = 2;
+int var3 = 3;
+string val = "value";
+string field1 = "field1";
+string field2 = "field2";
+boolean a = true;
+boolean b = false;
+boolean c = true;
+boolean condition = true;
+int trueVal = 1;
+int falseVal = 0;
+int value1 = 100;
+int value2 = 200;
+int x = 5;
+int age = 30;
+string title = "Book Title";
+xml xmlVar = xml `<root/>`;
+string[] args = ["arg1", "arg2"];
+string arg1 = "arg1";
+string arg2 = "arg2";
+Person parent = {'field: "value", field1: "value1", field2: "value2"};
+Record rec = {field1: "value1", field2: "value2"};
```

Comment:
Don't we need to include a string template expression related variable here for testing?

Reply:
got it